metadata-service/services/src/main/java/com/linkedin/metadata/service/RollbackService.java

```diff
@@ -173,7 +173,7 @@ public RollbackResponse rollbackIngestion(
     // Rollback timeseries aspects
     DeleteAspectValuesResult timeseriesRollbackResult =
         timeseriesAspectService.rollbackTimeseriesAspects(opContext, runId);
-    rowsDeletedFromEntityDeletion += timeseriesRollbackResult.getNumDocsDeleted();
+    rowsDeletedFromEntityDeletion += timeseriesRollbackResult.getNumDocsDeleted().intValue();
```

Comment:
Included in PR : https://github.com/datahub-project/datahub/pull/10337/files/a92d1fa5a8b57e9275c03c996130d956e9c31fc8#diff-0b029d8cd44d3e3d5f46d04a9ec12bb6a0d4cab6800663b45cfacb135bb51301